SphinxDocs/import.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+cd "$(dirname "$0")"
+set -e
+
+root_html_doc_dir='../Doc/Manual'
+sphinx_src_dir='source/Manual'
+
+mkdir -p $sphinx_src_dir
+
+for html in $root_html_doc_dir/*.html
+do
+    echo converting $html
+    html_base_name=$(basename $html)
+    rst_base_name=${html_base_name/html/rst}
+    rst=$sphinx_src_dir/$rst_base_name
+    pandoc -o $rst $html
```

Comment:
> I was wondering how you converted the hand written html files to rst format and how error prone this is?

I've used [pandoc](https://pandoc.org/). I've convert html documentation files to RST and html website to MD using pandoc.